src/gameState.ts

```diff
@@ -12,6 +13,22 @@ export interface Cell {
 
 export type StoneColor = "black" | "white";
 
+export function whoHasWon(gameState: GameState): StoneColor {
```

Comment:
I tend to prefer names that help to form a sentence when used in the code. 

e.g. `if(getWinner(gameState) === BLACK)` is more natural to read than `if(whoHasWon(gameState) === BLACK)`